clients/cloud/rust/README.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+Produce messages to produce and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the [rust-rdkafka client for Apache Kafka](https://github.com/fede1024/rust-rdkafka).
```

Comment:
"Produce messages to produce .." is redundant, should probably be "Produce messages to and consume messages from .."

Reply:
copy/paste from other client examples - I left it as is for uniformity